vumi/servicemaker.py

```diff
@@ -197,6 +203,12 @@ class VumiWorkerServiceMaker(object):
     # what command line options does this service expose
     options = StartWorkerOptions
 
+    def set_maxthreads(self, options):
```

Comment:
That method was originally doing something more complicated. I'll make the change, thanks.